utils/docker/Dockerfile.checkpatch

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2021 Intel Corporation
+# All rights reserved.
+#
+# 'recipe' for Docker to build a image for running checkpatch.
+#
+
+# Pull base image
+FROM centos:centos7
+
+ARG CACHEBUST
+
+# Install OS updates and package.  Include basic tools and daos dependencies
+# that come from the core repo.
+# Clean up any repos afterwards to save space.
+RUN yum -y install epel-release &&				\
+    yum -y upgrade &&						\
+    yum -y install file git python3-pip python36-clustershell	\
+	python36-numpy python36-paramiko python36-pylint	\
+	python36-pyxattr python36-requests python36-tabulate	\
+	ShellCheck &&						\
+    yum clean all
+
+ARG UID=1000
+
+# Add DAOS users
+RUN useradd --uid $UID --user-group --create-home --home /home/daos daos
+
+WORKDIR /home/daos
+USER daos:daos
+
+RUN pip3 --no-cache-dir --disable-pip-version-check install --user	\
+	avocado-framework				\
+	avocado-framework-plugin-result-html		\
+	avocado-framework-plugin-varianter-yaml-to-mux	\
+	codespell					\
+	pygithub
```

Comment:
All of this probably comes packaged in Fedora.

Reply:
pygithub is packaged in centos7 already, however I found if was missing a "dismiss" function that we use so I stuck with the pip version.  For codespell I thought pip was probably more up-to-date than centos although this probably doesn't matter all that much.

Reply:
Did you check the version packaged in Fedora?